tools/cythonize.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I suppose this is safe now that we're dropping 2 - assuming 4 doesn't come along any time soon.